rubi/pyproject.toml

```diff
@@ -21,6 +21,8 @@ eth-utils = "2.1.0"
 subgrounds = { version = "1.6.0", extras = ["dash"] }
 pyyaml = "6.0.0"
 python-semantic-release = "7.34.3"
+pre-commit = "^3.3.3"
```

Comment:
Is it possible to make these `dev-dependencies`?

When running a bot in production these dependencies aren't needed.